client/src/pages/guide/english/css/css-buttons/index.md

```diff
@@ -102,6 +102,35 @@ button {
 }
 ```
 
+### Button Click Animation
+
+To animate a button on click use 'button:active':
+
+```
+.button {
```

Comment:
All the examples work on the element, not a class, as they should, so change accordingly.